src/features/draw-rolling-circle/Drawing.js

```diff
@@ -0,0 +1,108 @@
+// @ts-check
+
+import { CanvasBuilder } from '@/components/Canvas'
+import { Color, Physics } from '@/config'
+
+const HEIGHT = 100
+const WIDTH = 500
+const radius = 30
+let pointX, pointY
```

Comment:
Based on the code so far, I don't think we need to store these outside of the draw factory's / draw function's scope.

Down inside `draw`, you can 

```js
    // draw the line
    const pointX = rollingCircle.centerX + radius * Math.cos(rollingCircle.theta)
    const pointY = rollingCircle.centerY + radius * Math.sin(rollingCircle.theta)
```

Reply:
Cleaned up.